railgun/test/integration/suite_test.go

```diff
@@ -145,18 +146,28 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 
 		// if set, allow running the legacy controller for the tests instead of the current controller
 		var cmd *exec.Cmd
-		stdout, stderr := new(bytes.Buffer), new(bytes.Buffer)
 		if useLegacyKIC() {
 			cmd = buildLegacyCommand(ctx, kubeconfig.Name(), adminHost, cluster.Client())
+			stdout, stderr := new(bytes.Buffer), new(bytes.Buffer)
+			cmd.Stdout = io.MultiWriter(stdout, os.Stdout)
+			cmd.Stderr = io.MultiWriter(stderr, os.Stderr)
```

Comment:
@shaneutt shall we make legacy KIC print to stdout/stderr directly instead of buffering?

Reply:
The intention with the multiwriter was to do both, so that test logging can emit the stdout/stderr AND tests could inspect the output programmatically as well if desired.

Reply:
Isn't this an instance of YAGNI?

Reply:
Sure, we can drop it for now :+1: